app/Http/Controllers/CustomFieldsController.php

```diff
@@ -94,6 +94,7 @@ public function store(CustomFieldRequest $request)
             "field_encrypted" => $request->get("field_encrypted", 0),
             "show_in_email" => $request->get("show_in_email", 0),
             "is_unique" => $request->get("is_unique", 0),
+            "display_in_user_view" => $request->get("display_in_user_view", 0),
```

Comment:
For the other comment I made down below, this might be the place where I would make sure that we don't accidentally set 'show in email' or 'display in user view'. I might not be right about that though.

Reply:
I've added a few lines to override those fields if encrypted.